.github/workflows/cmake.yml

```diff
@@ -86,6 +89,7 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 90
     env:
+      APACHE_ARROW: ${{ matrix.apache-arrow }}
```

Comment:
Do we need this?

It seems that we can use `matrix.apache-arrow` directly instead of `env.APACHE_ARROW`.

Reply:
fix [97394ae](https://github.com/groonga/groonga/pull/2459/commits/97394ae7c5c8b522310cf43b1daf49ca661d591c): We don't need to define it comparing with the other values.